osu.Desktop.VisualTests/VisualTestGame.cs

```diff
@@ -14,7 +14,7 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            new BackgroundScreenDefault { Depth = 10 }.LoadAsync(this, AddInternal);
+            LoadComponentAsync(new BackgroundScreenDefault { Depth = 10 }, AddInternal);

```

Comment:
`LoadAsync` is protected now. do you not agree this hugely improves the readability of code? it means you don't need to get a game via DI to do async loading. also you may want to load via `Add` not `AddInternal`. or call `AddInternal` slightly later (as is the case for preloaded song select etc.)